rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -140,7 +140,9 @@ error_code sys_rsx_memory_free(cpu_thread& cpu, u32 mem_handle)
 		return CELL_ENOMEM;
 	}
 
-	if (rsx::get_current_renderer()->dma_address)
+	const auto render = rsx::get_current_renderer();
```

Comment:
What's going on here? This variable is never used again.

Reply:
Fixed.